task3.py

```diff
@@ -0,0 +1,55 @@
+from flask import Flask, render_template, request, redirect, url_for, flash
+from models import db, User
+from forms import LoginForm
+from flask_wtf.csrf import CSRFProtect
+from flask_bootstrap import Bootstrap5
+
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///userdatabase.db'
+db.init_app(app)
+app.config['SECRET_KEY'] = b'c928ba4f26979fe137232ae1a30cff74a2602ddeaf28ca226969724afacc5686'
+csrf =CSRFProtect(app)
+bootstrap = Bootstrap5(app)
+
+
+@app.cli.command('init-db')
+def init_db():
+     db.create_all()
+     print('ok')
+
+
+@app.route('/')
+def index():
+    context = {'title':'Welcom to MySite'}
+    return render_template('base.html', **context)
+
+
+@app.route('/reg/', methods=['GET','POST'])
+def reg_user():
+    form = LoginForm()
+    if request.method == 'POST' and form.validate_on_submit:
```

Comment:
`validate_on_submit` это метод, а не атрибут. То, как вы его написали всегда будет возвращать `True`
```suggestion
    if form.validate_on_submit():
```